nupic/research/frameworks/dynamic_sparse/models/loggers.py

```diff
@@ -0,0 +1,220 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
Should this be in the models directory, or in common or even loggers directory?

Reply:
I am also not sure. I've left there temporarily while I figure out a better place, I will take suggestions. Probably in common

Reply:
👍